src/test/java/gov/healthit/chpl/aqa/pageObjects/SurveillanceComplaintsPage.java

```diff
@@ -0,0 +1,117 @@
+package gov.healthit.chpl.aqa.pageObjects;
+
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+
+/**
+ * Class SurveillanceComplaintsPage definition.
+ */
+public final class SurveillanceComplaintsPage extends BasePage {
+    private SurveillanceComplaintsPage() {}
+
+    public static WebElement actions(final WebDriver driver) {
+        return driver.findElement(By.id("actions"));
+    }
+
+    public static WebElement addNewComplaintButton(final WebDriver driver) {
+        return driver.findElement(By.id("add-new-complaint"));
+    }
+
+    public static WebElement addComplaintFormTitle(final WebDriver driver) {
+        return driver.findElement(By.xpath("//*[@id=\"chpl-complaint\"]/div[1]/h2"));
+    }
+
+    public static WebElement associatedCriteriaValue(final WebDriver driver) {
+        return driver.findElement(By.xpath("//*[@id=\"chpl-complaint\"]/div[13]/div[1]/span/button"));
+    }
+
+    public static WebElement associatedListingValue(final WebDriver driver) {
+        return driver.findElement(By.xpath("//*[@id=\"chpl-complaint\"]/div[11]/div/span/button"));
+    }
+
+    public static WebElement associatedSurveillanceValue(final WebDriver driver) {
+        return driver.findElement(By.xpath("//*[@id=\"chpl-complaint\"]/div[12]/div/span/button"));
+    }
+
+    public static WebElement certifiedProductField(final WebDriver driver) {
+        return driver.findElement(By.id("listings"));
+    }
+
+    public static WebElement complainantContactedCheckbox(final WebDriver driver) {
+        return driver.findElement(By.id("complainant-contacted"));
+    }
+
+    public static WebElement complaintsTable(final WebDriver driver) {
+        return driver.findElement(By.id("complaints-table"));
+    }
+
+    public static WebElement complainantType(final WebDriver driver) {
+        return driver.findElement(By.id("complainant-type"));
+    }
+
+    public static WebElement closedDate(final WebDriver driver) {
+        return driver.findElement(By.id("closed-date"));
+    }
+
+    public static WebElement criteriaField(final WebDriver driver) {
+        return driver.findElement(By.id("criteria"));
+    }
+
+    public static WebElement deleteComplaintButton(final WebDriver driver) {
+        return driver.findElement(By.cssSelector("a[title='Delete Complaint']"));
+    }
+
+     public static WebElement developerContactedCheckbox(final WebDriver driver) {
+        return driver.findElement(By.id("developer-contacted"));
+    }
+
+    public static WebElement editComplaintButton(final WebDriver driver) {
+        return driver.findElement(By.xpath("//*[@id=\"main-content\"]/div/div/div/div[3]/table/tbody/tr[1]/td[7]/button[1]/i"));
```

Comment:
Recommend requesting some sort of ID for this, though not required

Reply:
Updated definition to use cssSelector, this was old definition that wasn't updated & wasn't used.